plugins/node/opentelemetry-instrumentation-aws-lambda/src/instrumentation.ts

```diff
@@ -433,32 +413,8 @@ export class AwsLambdaInstrumentation extends InstrumentationBase<AwsLambdaInstr
   private static _determineParent(
     event: any,
     context: Context,
-    disableAwsContextPropagation: boolean,
     eventContextExtractor: EventContextExtractor
   ): OtelContext {
-    let parent: OtelContext | undefined = undefined;
-    if (!disableAwsContextPropagation) {
-      const lambdaTraceHeader = process.env[traceContextEnvironmentKey];
```

Comment:
now that we removed the usage of `traceContextEnvironmentKey`, I think we also need to clean [this line](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/9a20e15547669450987b2bb7cab193f17e04ebb7/plugins/node/opentelemetry-instrumentation-aws-lambda/src/instrumentation.ts#L76) from the file:

```
export const traceContextEnvironmentKey = '_X_AMZN_TRACE_ID';
```